frontend/views/pages/Contributions.vue

```diff
@@ -216,8 +217,9 @@ export default {
       'groupSettings',
       'memberProfile'
     ]),
-    currency () {
-      return currencies[this.groupSettings.mincomeCurrency]
+    mincome () {
+      const settings = this.groupSettings
+      return currencies[settings.mincomeCurrency].displayWithCurrency(settings.mincomeAmount)
```

Comment:
we are using this a lot of times across the project, we could dry it a little with a vuex getter `groupMincomeFormatted`